instructor-guides/advanced_data_protection_INSTRUCTOR_GUIDE.md

```diff
@@ -0,0 +1,134 @@
+3. Include the following information in the description:
+    1. The name of the host (probably yourself) after "Host:"
+    2. The names of presenters after "Presenters:"
+    3. A list of tracks that your workshop will use after "Tracks:", listing the URL of each track on its own line.
+
+Before saving the event, be sure to set the calendar as "instruqt-workshops" instead of your own personal calendar.
+
+### Email Invitation
+Here is some boilerplate text you can use or customize when inviting or announcing your workshop:
+
+```
+Vault Advanced Data Protection
+A hands-on technical workshop
+
+Learn how to leverage Vault for encrypting sensitive customer data. Vault is an API-driven, cloud-agnostic secrets management & encryption solution. Vault is easy for beginners and powerful for experts. Join us for a half-day training that will get you up and running quickly with Vault.
+
+Topics covered in the workshop include:
+
+* Vault Overview
+* Interacting with Vault
+* Vault Architecture
+* Vault Secret Engines and Authentication Methods
+* Vault Policies
+* Dynamic Database Secrets
+* Encryption as a Service
+* Format Preserving Encryption
+* Tokenization
+* Filesystem & Database Encryption
+
+The only prerequisites for this workshop are a web browser and willingness to learn.
+```
+
+### Markdown Slide Deck
+The slide deck for this training is published here:
+
+#### https://hashicorp.github.io/field-workshops-vault/slides/multi-cloud/adp
+
+#### Navigation
+Use the arrow keys (up/down or left/right) to navigate back and forth between slides.
+
+Press the `P` key to enter presenter mode and reveal the speaker notes.
+
+Press the `C` key to pop open an external window that will stay synced with your speaker notes window. This is useful for keeping notes on your laptop while showing the presentation on the projector.
+
+#### RemarkJS
+The slide deck for this training is written completely in [Markdown](https://guides.github.com/features/mastering-markdown/) using the [RemarkJS](https://remarkjs.com/#1) framework. This allows us to publish slide decks from a source code repository. The slide deck is easy to maintain, lightweight, and can be accessed from anywhere. Updates and changes to the deck can be made quickly and efficiently by simply editing the markdown source files. If you find any mistakes or issues with the slides please add them to our issue tracker:
+
+https://github.com/hashicorp/field-workshops-vault/issues
+
+### Hands-on Labs
+At certain points in the slide deck there are links to the lab exercises. [Instruqt](https://instruqt.com/hashicorp) is our lab platform. While all of the Vault workshop tracks are actually public, the slides use links to permanent Instruqt invitations that require users to register with Instruqt. We do this so that HashiCorp Field Marketing can collect the emails of users who start tracks during public or private workshops or on their own after following the links from the slides.
```

Comment:
Please fix this. What is said about embedded invite links is true of the standard Vault workshop slides, but not of your slides.  I actually recommend creating permanent new invites for all 3 tracks.  I can do that if you want.  You can then embed links to those invites under nice text.  I could also do that for you.

Reply:
However, we might have to be careful about creating permanent links to some of these Vault tracks if they embed Vault license.

Reply:
I took care of this.